### DISCUSSION TITLE
Adding Defold?

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Extras#9970] [Q&A] [answered]
I just saw that there is some old pull request about adding Defold, which was not merged [here](https://github.com/ScoopInstaller/Extras/pull/4335) ,

Any ideas if the Defold is gonna be added now, since it is now in BETA?

**Comment by HUMORCE** [comment, marked as answer]
It already has a stable version, but with a Special publishing method, you can make a PR or a package request.
